### PR TITLE
Killed read-it-later, system-guide, and personas

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Personas ended up being quite the rabbit hole. Happy New Year, folks. :boom:

### CODE REVIEW

**Comment by ryanpbrewster** [comment]
I don't think there are going to be many merge conflicts with this, and it's not a high priority, so I'll leave this open for a few days. @andrewconner @leogrim @camhashemi

**Comment by andrewconner** [comment]
Luuucky

File: kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala [hunk 1]
```diff
@@ -134,77 +131,28 @@ class LDACommanderImpl @Inject() (
...
-          scoreAndWeightOpt match {
-            case Some((s, w)) => (Some(s), w)
-            case None => (None, 0f)
-          }
-        case None => (None, 0f)
-      }
+    uriTopicOpts.map { uriTopicOpt =>
+      if (!isInJunkTopic(uriTopicOpt, junkTopics)) scoreURI(uriTopicOpt, userFeatsCombo)
+      else LDAUserURIInterestScores(None, None, None)
     }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: @andrewconner could you take a quick once-over on this function.

Comment (andrewconner) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: Yep sure, will get to this in a few.

File: kifi-backend/modules/cortex/app/com/keepit/cortex/models/lda/LDAUserStatDbUpdater.scala [hunk 2]
```diff
@@ -124,8 +123,8 @@ class LDAUserStatDbUpdaterImpl @Inject() (
     def changedMuch(numOfEvidenceBefore: Int, numOfEvidenceNow: Int) = {
       val diff = abs(numOfEvidenceNow - numOfEvidenceBefore)
       if (model.get.state != UserLDAStatsStates.ACTIVE && numOfEvidenceNow >= min_num_evidence) true
-      else if (numOfEvidenceNow < min_num_evidence && model.get.state == UserLDAInterestsStates.ACTIVE) true
```
Comment (ryanpbrewster) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/models/lda/LDAUserStatDbUpdater.scala:127]: This was a straight-up bug, the types didn't match so they were never equal.

**Pull request merged**